projectDocs/dev/createDevEnvironment.md

```diff
@@ -26,35 +26,37 @@ git pull
-		* When you are intending to use the Visual Studio IDE (not required for NVDA development), you can download [the community version](https://aka.ms/vs/16/release/vs_Community.exe), which is also used by appveyor
-		* The Professional and Enterprise versions are also supported
-		* Preview versions are *not* supported
-	* When installing Visual Studio 2019, you need to enable the following:
-		* In the list on the Workloads tab
-			* in the Windows grouping:
-				* Desktop development with C++
-			* Then in the Installation details tree view, under Desktop for C++, Optional, ensure the following are selected:
-				* MSVC v142 - VS 2019 C++ x64/x86 build tools
-				* Windows 11 SDK (10.0.22000.0)
-				* C++ ATL for v142 build tools (x86 & x64)
-				* C++ Clang tools for Windows
-		* On the Individual components tab, ensure the following items are selected:
-			* MSVC v142 - VS 2019 C++ ARM64 build tools
-			* C++ ATL for v142 build tools (ARM64)
-	* If installing Visual Studio 2022: choose all the same above components as for 2019, but V143 variants, rather than V142. 
+#### Python
+[Python](https://www.python.org/), version 3.11, 32 bit.
+
+To replicate the production build environment, use the 3.11.x minor version of Python that [AppVeyor uses for the Visual Studio 2022 environment](https://www.appveyor.com/docs/windows-images-software/#python).
+
+#### Microsoft Visual Studio
+* Microsoft Visual Studio 2022
+	* To replicate the production build environment, use the [version of Visual Studio 2022 that AppVeyor is using](https://www.appveyor.com/docs/windows-images-software/#visual-studio-2022). 
+	* When you do not use the Visual Studio IDE itself, you can download the [build tools](https://aka.ms/vs/17/release/vs_BuildTools.exe)
+	* When you are intending to use the Visual Studio IDE (not required for NVDA development), you can download [the community version](https://aka.ms/vs/17/release/vs_Community.exe), which is also used by AppVeyor
+	* The Professional and Enterprise versions are also supported
+	* Preview versions are *not* supported
+* When installing Visual Studio, additional components must be included
+	* You can automatically fetch these using [NVDAs .vsconfig](../../.vsconfig) using the [import feature of the VS installer](https://learn.microsoft.com/en-us/visualstudio/install/import-export-installation-configurations?view=vs-2022#import-a-configuration)
+	* In the list on the Workloads tab, in the Desktop grouping:
+		* Desktop development with C++.
+			* Once selected, ensure "C++ Clang tools for Windows" is included under the optional grouping.
+	* On the Individual components tab, ensure the following items are selected:
+		* Windows 11 SDK (10.0.22621.0)
+		* MSVC v143 - VS 2022 C++ ARM64 build tools
+		* MSVC v143 - VS 2022 C++ x64/x86 build tools
+		* C++ ATL for v143 build tools (x86 & x64)
+		* C++ ATL for v143 build tools (ARM64)

```

Comment:
Can the ARM64EC label be added to enhance the accuracy of the document?

Reply:
```suggestion
		* MSVC v143 - VS 2022 C++ ARM64 build tools
		* MSVC v143 - VS 2022 C++ x64/x86 build tools
		* C++ ATL for v143 build tools (x86 & x64)
		* C++ ATL for v143 build tools (ARM64/ARM64EC)
```